lib/Hash/MoreUtils.pm

```diff
@@ -84,6 +91,14 @@ sub slice_exists
     return map { $_ => $href->{$_} } grep { exists($href->{$_}) } @list;
 }
 
+sub slice_delete
```

Comment:
For me it reads as a "slice_missing"

Reply:
huh ?

Reply:
The implementation is kind-of unfortunate ... please have a look at pr#5 and see the common spirit :)

Reply:
Ah ... s such

`slice_missing` creates `key => undef` pairs for those not in LIST
`slice_delete` deletes or removes existing (key/value) pairs if the key is in LIST (but without destroying the original)